configuration/inputs.md

```diff
@@ -233,10 +234,19 @@ A saved script can also be referenced directly via the command parameter \(see s
       script: |
         for i in {1..50}
           do
-            echo "Welcome $i times" 
+            echo "Welcome $i times"
           done
 ```
 
+## Kubernetes Events
+
+If enabled, agents will report the warning-level events generated by Kubernetes (namespace independent). Note that Kubernetes Events should be used with `ED_LEADER_ELECTION_ENABLED` environment variable in order to observe events.
```

Comment:
please replace "agents will report the warning-level events" with "agents will collect events"

no need to mention about our Warning filter that is applied in the API level. we actually collect all events

Reply:
Let's also mention about the env variable's value
`ED_LEADER_ELECTION_ENABLED=1`

Reply:
Done